### PR TITLE
Translated tags.js & describe.js into Korean

### PR DESCRIPTION
**Pull request by JuhoKang**
also fixed indentations you mentioned.

### CODE REVIEW

File: src/levels/mixed/describe.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-exports.level = {
+﻿exports.level = {
```
Comment (pcottle) [inline src/levels/mixed/describe.js]: Hrm what is the change here? hard to tell but it looks like whitespace?

File: src/levels/rampup/cherryPick.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ exports.level = {
     "zh_CN": "git cherry-pick 后面跟着 commit 的名字",
     "zh_TW": "git cherry-pick 後面要接著 commit 的名稱",
     "ru_RU": "git cherry-pick основывается на именах коммитов!",
-	"ko": "커밋의 이름들로 git cherry-pick 하세요!"
+    "ko"   : "커밋의 이름들로 git cherry-pick 하세요!"
```
Comment (pcottle) [inline src/levels/rampup/cherryPick.js]: thanks!

**Comment by pcottle** [comment]
Great! Little weird that the first line of each file shows a change -- maybe its just whitespace? the tests pass though so ill trust you :tada: :+1:

**Comment by JuhoKang** [comment]
I'll find out what the "weird invisible change" is next time I work on it.
Maybe it's because of the text editor I used. (notepad++ portable) 
The indentation problem was caused by this editor so the "weird invisible change" might also be

**Pull request merged**